components/host-dashboard/HostDashboardReports.js

```diff
@@ -165,11 +166,15 @@ const getDefaultDateInterval = () => {
   };
 };
 
-const HostDashboardReports = ({ hostSlug }) => {
+const HostDashboardReports = ({ hostSlug, hostCreateDate }) => {
   const [dateInterval, setDateInterval] = React.useState(getDefaultDateInterval);
   const [collectives, setCollectives] = React.useState(null);
-  const dateFrom = simpleDateToISOString(dateInterval?.from, false, dateInterval?.timezoneType);
-  const dateTo = simpleDateToISOString(dateInterval?.to, true, dateInterval?.timezoneType);
+  const dateFrom = simpleDateToISOString(dateInterval?.from || hostCreateDate, false, dateInterval?.timezoneType);
+  const dateTo = simpleDateToISOString(
+    dateInterval?.to || dayjs().startOf('day').toISOString(),
+    false,
+    dateInterval?.timezoneType,
+  );
```

Comment:
Why not, I think the timezone of `dayjs().startOf('day').toISOString()` is not properly managed. Can you double check this? Also, doesn't make sense to me to be the beginning of current day, as we will be missing transactions done since the beginning of the day. End of day would have better results.

Reply:
In the end, I think it might be better to work on this on the API side.